src/pages/AddNewCake.js

```diff
@@ -0,0 +1,34 @@
+import "./AddNewCake.css";
+
+export default function AddNewCake() {
+  return (
+    <form className="form">
```

Comment:
This class Name seems redundant

Reply:
It seems I was in a hurry there. The naming is corrected. :)